### ISSUE TITLE
Ubuntu 18.04 with PPA SAB: traceback "UnicodeDecodeError: 'ascii' codec can't decode byte 0xef in position 8593: ordinal not in range(128)"

### ISSUE THREAD

**Issue by sanderjo** [sabnzbd/sabnzbd#1787]
Fresh Ubuntu 18.04 (in a docker), SABnzbd from PPA, gives a traceback. The SAB wizard goes well, but then first access gives the traceback.

Ubuntu start:
```
docker run -p 8080:8080 -it ubuntu:18.04 /bin/bash
```

In the Ubuntu container:
```
apt update
apt-get install software-properties-common -y
add-apt-repository ppa:jcfp/ppa
apt update
apt upgrade
apt-get install sabnzbdplus
sabnzbdplus -b0 -l2 -s 0.0.0.0
```

stdout when accessing from Chrome (and happens too with `curl -v http://127.0.0.1:8080/'):

```
2021-02-06 19:11:48,067::DEBUG::[config:732] Missing configuration item servers,adsf
2021-02-06 19:11:48,073::INFO::[config:905] Writing settings to INI file /root/.sabnzbd/sabnzbd.ini
2021-02-06 19:11:49,487::INFO::[config:905] Writing settings to INI file /root/.sabnzbd/sabnzbd.ini
2021-02-06 19:11:49,570::INFO::[notifier:122] Sending notification: Error - [06/Feb/2021:19:11:49] HTTP  (type=error, job_cat=None)
2021-02-06 19:11:49,570::ERROR::[_cplogging:219] [06/Feb/2021:19:11:49] HTTP 
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/cherrypy/_cprequest.py", line 670, in respond
    response.body = self.handler()
  File "/usr/lib/python3/dist-packages/cherrypy/lib/encoding.py", line 220, in __call__
    self.body = self.oldhandler(*args, **kwargs)
  File "/usr/lib/python3/dist-packages/cherrypy/_cpdispatch.py", line 60, in __call__
    return self.callable(*self.args, **self.kwargs)
  File "/usr/share/sabnzbdplus/sabnzbd/interface.py", line 123, in internal_wrap
    return wrap_func(*args, **kwargs)
  File "/usr/share/sabnzbdplus/sabnzbd/interface.py", line 404, in index
    return template.respond()
  File "_usr_share_sabnzbdplus_interfaces_Glitter_templates_main_tmpl.py", line 452, in respond
  File "/usr/lib/python3/dist-packages/Cheetah/Template.py", line 1703, in _handleCheetahInclude
    self.getFileContents(path)
  File "/usr/lib/python3/dist-packages/Cheetah/Template.py", line 1506, in getFileContents
    output = fp.read()
  File "/usr/lib/python3.6/encodings/ascii.py", line 26, in decode
    return codecs.ascii_decode(input, self.errors)[0]
UnicodeDecodeError: 'ascii' codec can't decode byte 0xef in position 8593: ordinal not in range(128)
2021-02-06 19:11:49,573::INFO::[_cplogging:219] [06/Feb/2021:19:11:49] HTTP 
Request Headers:
  Remote-Addr: 172.17.0.1
  HOST: 127.0.0.1:8080
  CONNECTION: keep-alive
  UPGRADE-INSECURE-REQUESTS: 1
  USER-AGENT: Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/88.0.4324.150 Safari/537.36
  ACCEPT: text/html,application/xhtml+xml,application/xml;q=0.9,image/avif,image/webp,image/apng,*/*;q=0.8,application/signed-exchange;v=b3;q=0.9
  SEC-FETCH-SITE: same-origin
  SEC-FETCH-MODE: navigate
  SEC-FETCH-USER: ?1
  SEC-FETCH-DEST: document
  REFERER: http://127.0.0.1:8080/sabnzbd/wizard/two
  ACCEPT-ENCODING: gzip, deflate, br
  ACCEPT-LANGUAGE: en-NL,en;q=0.9,nl-NL;q=0.8,nl;q=0.7,en-US;q=0.6
  COOKIE: plushContainerWidth=100%25; plushMultiOps=1; plushNoTopMenu=0; plushContainerWidth=100%25; plushMultiOps=1; plushNoTopMenu=0
```

```
# cat /etc/lsb-release 
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=18.04
DISTRIB_CODENAME=bionic
DISTRIB_DESCRIPTION="Ubuntu 18.04.5 LTS"
```

**Comment by sanderjo** [comment]
Ah, it's a Glitter problem: when I change

`web_dir = Glitter`

to

`web_dir = Plush`

... I can use SABnzbd again.

**Comment by sanderjo** [comment]
Happens too with SAB 3.1.0, so it's not a recently introduced problem.

**Comment by Safihre** [comment]
@jcfp maybe you know why? 

Because we run the functional tests also on Python 3.6 so that can't be the problem..

**Comment by sanderjo** [comment]
> Because we run the functional tests also on Python 3.6 so that can't be the problem..

Functional tests against the webinterface with Glitter?

**Comment by sanderjo** [comment]
Triggered by advice from @Safihre  

Running from git-version of SAB ... works flawlessly ...

Weird.

**Comment by sanderjo** [comment]
FWIW

```
# diff -qr /git/sabnzbd/interfaces/Glitter/ /usr/share/sabnzbdplus/interfaces/Glitter/
Only in /git/sabnzbd/interfaces/Glitter/: licenses
Files /git/sabnzbd/interfaces/Glitter/templates/include_overlays.tmpl and /usr/share/sabnzbdplus/interfaces/Glitter/templates/include_overlays.tmpl differ
Files /git/sabnzbd/interfaces/Glitter/templates/main.tmpl and /usr/share/sabnzbdplus/interfaces/Glitter/templates/main.tmpl differ
Files /git/sabnzbd/interfaces/Glitter/templates/static/bootstrap/css/bootstrap.min.css and /usr/share/sabnzbdplus/interfaces/Glitter/templates/static/bootstrap/css/bootstrap.min.css differ
Files /git/sabnzbd/interfaces/Glitter/templates/static/bootstrap/fonts/glyphicons-halflings-regular.eot and /usr/share/sabnzbdplus/interfaces/Glitter/templates/static/bootstrap/fonts/glyphicons-halflings-regular.eot differ
Files /git/sabnzbd/interfaces/Glitter/templates/static/bootstrap/fonts/glyphicons-halflings-regular.svg and /usr/share/sabnzbdplus/interfaces/Glitter/templates/static/bootstrap/fonts/glyphicons-halflings-regular.svg differ
Files /git/sabnzbd/interfaces/Glitter/templates/static/bootstrap/fonts/glyphicons-halflings-regular.ttf and /usr/share/sabnzbdplus/interfaces/Glitter/templates/static/bootstrap/fonts/glyphicons-halflings-regular.ttf differ
Only in /usr/share/sabnzbdplus/interfaces/Glitter/templates/static/bootstrap/fonts: glyphicons-halflings-regular.woff
Only in /usr/share/sabnzbdplus/interfaces/Glitter/templates/static/bootstrap/fonts: glyphicons-halflings-regular.woff2
Files /git/sabnzbd/interfaces/Glitter/templates/static/bootstrap/js/bootstrap.min.js and /usr/share/sabnzbdplus/interfaces/Glitter/templates/static/bootstrap/js/bootstrap.min.js differ
Files /git/sabnzbd/interfaces/Glitter/templates/static/javascripts/jquery-3.5.1.min.js and /usr/share/sabnzbdplus/interfaces/Glitter/templates/static/javascripts/jquery-3.5.1.min.js differ
Files /git/sabnzbd/interfaces/Glitter/templates/static/javascripts/jquery-ui.min.js and /usr/share/sabnzbdplus/interfaces/Glitter/templates/static/javascripts/jquery-ui.min.js differ
Files /git/sabnzbd/interfaces/Glitter/templates/static/javascripts/moment-2.26.0.min.js and /usr/share/sabnzbdplus/interfaces/Glitter/templates/static/javascripts/moment-2.26.0.min.js differ
Only in /git/sabnzbd/interfaces/Glitter/templates/static/stylesheets/colorschemes: Auto.css
Only in /usr/share/sabnzbdplus/interfaces/Glitter/templates/static/stylesheets/colorschemes: Default.css
Only in /git/sabnzbd/interfaces/Glitter/templates/static/stylesheets/colorschemes: Light.css
Files /git/sabnzbd/interfaces/Glitter/templates/static/stylesheets/glitter.css and /usr/share/sabnzbdplus/interfaces/Glitter/templates/static/stylesheets/glitter.css differ
```

So I copied the Glitter template stuff from SABnzbd-3.2.0RC1 into /usr/share/sabnzbdplus/interfaces/ like this
```
wget https://github.com/sabnzbd/sabnzbd/releases/download/3.2.0RC1/SABnzbd-3.2.0RC1-src.tar.gz
tar xzvf SABnzbd-3.2.0RC1-src.tar.gz 
cd SABnzbd-3.2.0RC1/interfaces/
mv /usr/share/sabnzbdplus/interfaces/Glitter/ /usr/share/sabnzbdplus/interfaces/git-not-good-Glitter/
cp -avr Glitter/ /usr/share/sabnzbdplus/interfaces/
```
... and now the ppa's sabnzbd is running well.

**Comment by jcfp** [comment]
This is a combination of a non-UTF8 environment and an issue with the minified packaged jquery symlinked in the packages. It seems I inadvertently reverted the fix I made earlier, see [#1336](https://github.com/sabnzbd/sabnzbd/issues/1336#issuecomment-554095620). I'll restore that fix.

**Comment by sanderjo** [comment]
> This is a combination of a non-UTF8 environment and an issue with the minified packaged jquery symlinked in the packages. It seems I inadvertently reverted the fix I made earlier, see [#1336](https://github.com/sabnzbd/sabnzbd/issues/1336#issuecomment-554095620). I'll restore that fix.

Cool ... now it works again. Thanks.
